hnn_core/network.py

```diff
@@ -25,6 +25,8 @@ def read_spikes(fname, gid_dict=None):
         containing the range of Cell or input IDs of different
         cell or input types. If None, each spike file must contain
         a 3rd column for spike type.
+    tstart, tstop : float | None
+        Values defining the start and stop times of all trials.
```

Comment:
can you also say that this is only relevant for legacy files? You can also add a section below "Returns", call it "Notes" and describe the file format as in here: https://github.com/mne-tools/mne-python/blob/master/mne/channels/montage.py#L480-L521

Reply:
Thanks for linking this, I'll hold off on describing the file format until we've settled on a consistent way to store fixed variables. I do like the way the comment lines are implemented here though, especially since it allows us to add more information in future PR's without too many nightmares with legacy format.